src/open_inwoner/cms/cases/tests/test_htmx.py

```diff
@@ -441,13 +442,9 @@ def test_cases(self, m):
                 }
             ],
         )
-        submit_button = upload_form.get_by_role("button", name=_("Document uploaden"))
+        submit_button = upload_form.get_by_role("button", name=_("Upload documenten"))
```

Comment:
This seems to be assuming that the user can upload multiple documents, contrary to the description of the ticket. Is this intended?

Reply:
I'm leaving this as I think it's a minor issue and will be correct once multiple support has landed.